### DISCUSSION TITLE
Remove support for `GitVersion.Commandline` && `GitVersion.Portable`

### DISCUSSION THREAD

**Discussion by arturcic** [GitTools/GitVersion#2894] [Ideas]
Remove support for `GitVersion.Commandline` && `GitVersion.Portable` in favor of the tar.gz/zip files we generate at each release, and available on the release page.

I suggest we remove the support for these 2 or at least `GitVersion.Commandline` as anyway both embed the exact same executable as the one from tar.gz/zip archives.

Anyone that uses these 2 can switch to use the archives instead.

The reason is to simplify the build infrastructure and reduce the duplicates. 

@asbjornu @gep13 Ideas?

**Comment by asbjornu** [comment]
I've never understood the difference between `GitVersion.CommandLine` and `GitVersion.Portable` so I've always used the former. Can we deprecate `GitVersion.Portable` and refer people to `GitVersion.CommandLine` instead?

**Reply by arturcic** [reply]
Well honestly I would refer people to use the zip archive in this case, as both `GitVersion.CommandLine` and `GitVersion.Portable` use the archive version, just a different delivery channel - via nuget.org and chocolatey.

**Reply by asbjornu** [reply]
The problem I see with the ZIP archive is that it's not immediately executable.

**Reply by gep13** [reply]
This would be my concern as well.  The Chocolatey package sets up the shim to allow for immediate execution once installed.

**Reply by arturcic** [reply]
Ok then, we keep them.
